<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSelectTaxDep</class>
 <widget class="QDialog" name="DlgSelectTaxDep">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>302</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Card amount:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leCard"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Partner tax code</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lePartnerTaxCode"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnOK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
